aineko/core/node.py

```diff
@@ -111,28 +113,39 @@ def setup_datasets(
             pipeline: name of the pipeline
             inputs: list of dataset names for the inputs to the node
             outputs: list of dataset names for the outputs of the node
+            prefix: prefix for topic name (<prefix>.<dataset_name>)
```

Comment:
```suggestion
            prefix: prefix for topic name (`<prefix>.<dataset_name>`)
```
For some reason VS Code doesn't render `(<prefix>.<dataset_name>)` correctly.
<img width="814" alt="image" src="https://github.com/aineko-dev/aineko/assets/31799121/9becd3b7-6e52-4658-b047-2e20a05953a9">